### PR TITLE
fix and test make install target

### PR DESCRIPTION
**Pull request by ClausKlein**
add more git ignore entries too

The install target does not work #189

### CODE REVIEW

File: tao/x11/taox11.mpc [hunk 1]
```diff
@@ -160,7 +160,7 @@ project(taox11) : taox11_defaults, taolib {
     anyinsert_adapter.h
     CurrentC.h
     orb_typesC.h
-    InterfaceDefC.h
+//     InterfaceDefC.h
```
Comment (jwillemsen) [inline tao/x11/taox11.mpc:163]: Please remove files that don't exist instead of commenting them out

**Review by jwillemsen — CHANGES_REQUESTED** [review]
Please remove the files that don't exist anymore instead of commenting them out

File: tao/x11/taox11.mpc (hunk 1 above)
Comment (ClausKlein) [reply tao/x11/taox11.mpc:163]: ok

File: .gitignore [hunk 2]
```diff
@@ -35,3 +36,48 @@ bin/MPC/config/MPC.cfg*
 tao/x11/ifr_client/taox11_ifr_client_skel_export.h
 tao/x11/ifr_client/taox11_ifr_client_stub_export.h
 ridlbe/c++11/orb.pidlc
+examples/log_module/output
```
Comment (jwillemsen) [inline .gitignore:39]: At some point we should migrate to a .gitignore for each specific test

**Comment by ClausKlein** [comment]
# Note:

## there are still many problems:

### the ACE header will be not installed?

### the tao header will be only partly installed?

```
klein_cl@WD003223:~/Workspace/cpp/axcioma/stage/include/tao$ find . -name x11
./tao/x11
./x11
klein_cl@WD003223:~/Workspace/cpp/axcioma/stage/include/tao$ find . -name x11 | xargs ls -l
./tao/x11:
total 0
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 18 11:18 dynamic_any

./x11:
total 0
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 bidir_giop
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 codecfactory
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 dynamic_any
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 ior_interceptor
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 ior_table
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 18 11:18 logger
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 messaging
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 ort
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 pi
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 pi_server
drwxr-xr-x 1 klein_cl klein_cl 512 Jan 19 09:40 typecodefactory
klein_cl@WD003223:~/Workspace/cpp/axcioma/stage/include/tao$
```


### the pidl files are installed at a stange location?

```
klein_cl@WD003223:~/Workspace/cpp/axcioma/stage/include/tao$ find . -name DynamicAny
./DynamicAny
./x11/dynamic_any/home/klein_cl/Workspace/cpp/axcioma/ACE/TAO/tao/DynamicAny
klein_cl@WD003223:~/Workspace/cpp/axcioma/stage/include/tao$ find . -name DynamicAny | xargs tree
./DynamicAny
├── DynAnyFactory.h
├── DynAnyUtils_T.cpp
├── DynAnyUtils_T.h
├── DynAny_i.h
├── DynArray_i.h
├── DynCommon.h
├── DynEnum_i.h
├── DynSequence_i.h
├── DynStruct_i.h
├── DynUnion_i.h
├── DynValueBox_i.h
├── DynValueCommon_i.h
├── DynValue_i.h
├── DynamicAny.h
├── DynamicAny.pidl
├── DynamicAnyC.h
├── DynamicAnyS.h
└── dynamicany_export.h
./x11/dynamic_any/home/klein_cl/Workspace/cpp/axcioma/ACE/TAO/tao/DynamicAny
└── DynamicAny.pidl

0 directories, 19 files
klein_cl@WD003223:~/Workspace/cpp/axcioma/stage/include/tao$
```

**Comment by jwillemsen** [comment]
The install feature has never been tested and is not supported for AXCIOMA/TAOX11. We currently don't have plans to add an install feature, see https://www.remedy.nl/services/opensource-support.html for our commercial services through which we could add this for you.

**Comment by ClausKlein** [comment]
> The install feature has never been tested and is not supported for AXCIOMA/TAOX11. We currently don't have plans to add an install feature, see https://www.remedy.nl/services/opensource-support.html for our commercial services through which we could add this for you.

How should someone use the **SW** if it is not installable?

We normaly use `CMake `projects!
Can you explain, how should I use the idl compiler in our `CmakeLists.txt` if the `taox11 SW` and its dependencies are not installed?

**Pull request merged**